ip_validation.py

```diff
@@ -0,0 +1,29 @@
+import socket
+import re
+
+input_ip = input("Insert your ip: ")
+
+
+def check_valid_ip_socket(ip):
+    try:
+        socket.inet_aton(ip)
+        status_ip = True
+    except socket.error:
+        status_ip = False
+    return print("Status IP: ", status_ip)
+
+
+def check_valid_ip_re(ip):

```

Comment:
It looks like a working workaround but let's think about a more sophisticated regex.